lib/ansible/modules/network/iosxr/iosxr_interface.py

```diff
@@ -360,7 +360,7 @@ def main():
         enabled=dict(default=True, type='bool'),
         tx_rate=dict(),
         rx_rate=dict(),
-        delay=dict(default=10, type='int'),
+        delay=dict(default=5, type='int'),
```

Comment:
Why is this changed to 5 sec? As per spec it should default 10 secs for all platforms